ansibleinviewer/parser.py

```diff
@@ -0,0 +1,147 @@
+import argparse
+from typing import List, Tuple
+
+
+class Parser:
+
+    def __init__(self):
+        self._groups = None
+        self._no_groups = None
+        self._variables = None
+        self._no_variables = None
+        self._parser = argparse.ArgumentParser()
+        self._parse_arguments()
+
+    def _parse_arguments(self):
+        self._parser.add_argument(
+            '-i',
+            '--inventory',
+            required=True,
+            help='Path to the ansible inventory file'
+        )
+        self._parser.add_argument(
+            '-g',
+            '--groups',
+            default=None,
+            help='Groups to connect with'
+        )
+        self._group.add_argument(
+            '--hosts',
```

Comment:
I moved it to separate file, because it already took a lot of space in main.py and in my opinion it was hard to read - especially after adding arguments for variables.
Also, if in the future there'll be more possible arguments then argument parsing will take even more space in main script.
But if you feel that you want it to stay in main.py then I'll move it back there.
Or maybe you'd prefer for example separate file with functions for parsing arguments, but no enclosed in class - just bare functions imported in main.py?

Reply:
Moving it to a separate file is good. I just think that it does not need to be a new class wrapping the argparse object, as it does not add any value.

Reply:
Done